staking_deposit/settings.py

```diff
@@ -1,7 +1,7 @@
 from typing import Dict, NamedTuple
-from .version import __version__
+from eth_utils import decode_hex
 
-DEPOSIT_CLI_VERSION = __version__
+DEPOSIT_CLI_VERSION = '2.7.0'
```

Comment:
The __version__ metadata s controlled by release-please so don't change that

Reply:
Solved